docker-cadg-apache/Dockerfile

```diff
@@ -0,0 +1,18 @@
+### Create the AOI Apache web frontend image
+
+# Get base apache image from Docker Hub
+FROM httpd:2.4
+
+# Install VIM editor for configuration changes inside the machine 
+# This is not the recommended way to do it, its best to configure the file first.
+# However, it may be necessary to modify depending on the server.
+RUN apt-get update
+RUN apt-get install vim -y
```

Comment:
and nano, please!

Reply:
No nano for you! 


...okay i added it